CONTRIBUTING_feedback_and_requests.md

```diff
@@ -0,0 +1,71 @@
+# Contributing ideas, feedback, and requests
+
+The WinUI team greatly values public feedback.  We combine this feedback with our own internal strategy goals, weighing it alongside our existing commitments, resources, and private partnerships, to deliver maximum value for you.
+
+The guide below outlines how you can formulate this feedback for maximum effect and influence on WinUI's direction.
+
+## Phrasing constructively
+
+When experiencing a pain-point, it can be natural to focus on the negatives and resolving things from the perspective of the negative. However, telling the team what you'd like to *prevent* is less helpful and actionable than telling the team what you'd like to *achieve*. 
+
+Your feedback is most effective when it is a constructive call to action on the team, and is clear and detailed – especially on the "why" so that we can make sure whatever it is that we arrive at together appropriately focuses on your goal and your intended outcome from start to finish. 
+
+
+**Examples of constructively phrased feedback:**
+
+Instead of:
+
+  - The state of platform is disappointing. I am not going to consider WinUI until my trust has been earned.
+
+Try this:
+  - Deprecation of past frameworks has left me burned. The following would go a long way in earning my trust because my company is trying to launch an app next year and will need it supported for at least 5 more: 
+    - OSS
+    - High-confidence 5-year roadmap
+    - Guaranteed support timeline
+
+    This feedback provides clear actionable items that the WinUI team can take into consideration and address. 
+    
+    
+## Sample areas where your feedback can have large impact
```

Comment:
What about mentioning the https://github.com/microsoft/microsoft-ui-xaml-specs repository in here to as part of feedback channel for new features? Adding feedback on new controls here definitely helps shape the API and create usable controls for everyone.

Reply:
Great idea! I'm going to add this in under the features section.